File: RedSheet.DbEntities/Models/SecurityQuestion.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("SecurityQuestions", Schema = "dbo")]
	[Serializable]
	public partial class SecurityQuestion  
	{
 
	[Required]
	public bool Active { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int SecurityQuestionId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string SecurityQuestionName { get; set; } 

	public SecurityQuestion()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/security-question.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class SecurityQuestion {
    constructor(securityQuestion?: SecurityQuestion )  {
        let properties = [ "active", "securityQuestionId", "securityQuestionName",];
        for (let property of properties)
            if (securityQuestion && securityQuestion[property])
                this[property] = securityQuestion[property];
    }
 
	active : boolean = false ;
 
	securityQuestionId : number =   0 ;
 
    @required()
    @maxLength(50)
	securityQuestionName : string =   undefined;


}
